proc_dash/app.py

```diff
@@ -8,28 +8,37 @@
 from dash.dependencies import Input, Output, State
 from dash.exceptions import PreventUpdate
 
+import proc_dash.plotting as plot
 import proc_dash.utility as util
 from dash import Dash, ctx, dash_table, dcc, html
 
-app = Dash(
-    __name__,
-    external_stylesheets=["https://codepen.io/chriddyp/pen/bWLwgP.css"],
-)
+EMPTY_FIGURE_PROPS = {"data": [], "layout": {}, "frames": []}
+
+PIPE_COMPLETE_STATUS_SHORT_DESC = {
```

Comment:
I think this probably fits better in `utility.py` because that's where it'll be used.